lua/core/dashboard.lua

```diff
@@ -69,14 +69,22 @@ M.setup = function()
 
   vim.g.dashboard_session_directory = lvim.builtin.dashboard.session_directory
 
-  vim.cmd "let packages = len(globpath('~/.local/share/lunarvim/site/pack/packer/start', '*', 0, 1))"
+  local lv_site = "lunarvim.org"
+  local lv_path = "$HOME/.local/share/lunarvim"
 
-  vim.api.nvim_exec(
-    [[
-    let g:dashboard_custom_footer = ['LunarVim loaded '..packages..' plugins  ']
-]],
-    false
-  )
+  local lv_ver_full = vim.fn.system("git -C " .. lv_path .. "/lvim describe --tag")
```

Comment:
For the time being it's fine, but I wonder if we should not have this in a separate file (possibly autogenerated)

Reply:
Where do you think is proper place to have this?
Would you explain more about autogeneration?

Reply:
A workflow could generate a version file on commits on master, for example.

Reply:
I added a bootstrap function that returns LunarVim version instead.
I may be wrong but thought file I/O on every startup could be more expensive.